src/middlewares/ParamsChecker.js

```diff
@@ -26,7 +26,7 @@ class ParamsChecker {
       }
       return next();
     };
-    if (userId) checkParam(userId);
+    if (commentId) checkParam(commentId);
```

Comment:
I'll prefer if we stop using this short return method. Please always wrap this in parenthesis.

Reply:
Ok, will do.